admin/src/pages/View/components/NavigationManager/Create/index.tsx

```diff
@@ -0,0 +1,71 @@
+// @ts-ignore
+import { Button } from "@strapi/design-system/Button";
+import React, { useCallback, useMemo } from "react";
+import { useIntl } from "react-intl";
+import { getMessage } from "../../../../../utils";
+import { Form, validationSchemaFactory } from "../Form";
+import {
+  CommonProps,
+  CreateState,
+  FooterActionsFactory,
+  Navigation,
+} from "../types";
+
+interface Props extends CreateState, CommonProps {}
+
+export const INITIAL_NAVIGATION = {
+  name: "Navigation",
+  items: [],
+  visible: true,
+} as unknown as Navigation;
+
+export const Create = ({
+  setState,
+  current,
+  isLoading,
+  alreadyUsedNames,
+}: Props) => {
+  const { formatMessage } = useIntl();
+  const onSubmit = useCallback(
+    (updated: Navigation) => {
+      setState({
+        view: "CREATE",
+        current: updated,
+        alreadyUsedNames,
+      });
+    },
+    [setState]
+  );
+  const validationSchema = useMemo(
+    () => validationSchemaFactory(alreadyUsedNames, formatMessage),
+    [alreadyUsedNames]
+  );
```

Comment:
Add space between single blocks of code (consts, function) to increase readability